docs/jwst/outlier_detection/arguments.rst

```diff
@@ -37,12 +37,20 @@ that control the behavior of the processing:
   valid values go from 0.0 to 1.0.
 
 ``--snr`` (string, default='4.0 3.0')
-  The signal-to-noise values to use for bad pixel identification. Valid
-  values are a pair of floating-point values in a single string.
+  The signal-to-noise values to use for bad pixel identification.
+  Two values are specified, where the first value is used for rejecting outliers
+  in all pixels across the image, while the second value is used to reject
+  additional lower-level outliers in pixels that are adjacent to those
+  identified in the first pass.  Valid values are a pair of floating-point
```

Comment:
I kind of like the additional info provided in the astrodrizzle docs that gives more of a "why" explanation for the 2nd value: "Since cosmic rays often extend across several pixels, the adjacent pixels make use of a slightly lower SNR threshold". Perhaps include this additional sentence here. See [this page](https://hst-docs.stsci.edu/drizzpac/chapter-5-drizzlepac-software-package/5-2-astrodrizzle-the-new-drizzle-workhorse#id-5.2AstroDrizzleTheNewDrizzleWorkhorse-CreateaCosmicRayMaskforEachImage)

Reply:
Done.